public/app/features/explore/Logs/LogsTable.tsx

```diff
@@ -67,6 +74,27 @@ export function LogsTable(props: Props) {
       });
       // `getLinks` and `applyFieldOverrides` are taken from TableContainer.tsx
       for (const field of frameWithOverrides.fields) {
+        const cellOptions = {
+          type: TableCellDisplayMode.ColorBackground,
+          applyToRow: true,
+          mode: TableCellBackgroundDisplayMode.Basic,
+        };
+
+        const initialDisplay = field.display;
+
+        const display: DisplayProcessor = (value, decimals) => {
+          if (value && typeof value === 'string' && Object.values(LogLevel).includes(value)) {
+            return {
+              color: getLogLevelColor(config.theme2, value),
+              text: value,
+            };
+          }
+          if (initialDisplay) {
+            return initialDisplay(value, decimals);
+          }
+          return undefined;
+        };
```

Comment:
rather than overring and setting a custom display processor defining a simple value mapping would do the trick (allows mapping values to colors)

Reply:
and with that I mean the standard field.confing.mappings (standard field config option)